ledger/ledger.go

```diff
@@ -462,11 +462,25 @@ func (l *Ledger) notifyCommit(r basics.Round) basics.Round {
 	if l.archival {
 		// Do not forget any blocks.
 		minToSave = 0
+	} else if minCatchpointsRoundsLookback := l.calcMinCatchpointRoundsLookback(); minCatchpointsRoundsLookback > 0 {
+		catchpointsMinToSave := r.SubSaturate(minCatchpointsRoundsLookback)
+		if catchpointsMinToSave < minToSave {
+			minToSave = catchpointsMinToSave
+		}
 	}
```

Comment:
Don't need the `if` (when `minCatchpointsRoundsLookback`==0 there's no effect), so don't need the intermediate var.
```suggestion
	} else {
	    catchpointsMinToSave := r.SubSaturate(l.calcMinCatchpointRoundsLookback())
		if catchpointsMinToSave < minToSave {
			minToSave = catchpointsMinToSave
		}
	}
```

Reply:
my patch probably has bad tabs/spaces but it easier to type than describe it

Reply:
I like the suggested code better

Reply:
Agreed, addressed in latest commit.